app/errors.py

```diff
@@ -0,0 +1,18 @@
+class VaccineError(Exception):
+    """Parent exception for all vaccine-related issues."""
+    pass
+
+
+class NotVaccinatedError(VaccineError):
+    """Raised when a visitor is not vaccinated."""
+    pass
+
+
+class OutdatedVaccineError(VaccineError):
+    """Raised when a visitor's vaccine is expired."""
+    pass
```

Comment:
The checklist suggests that you should use descriptive error messages. Currently, your custom exceptions do not provide any error messages.